test/fw/ptl/utils/pbs_snaputils.py

```diff
@@ -1137,8 +1138,15 @@ def __evaluate_core_file(self, file_path, core_dir):
             self.logger.debug("Could not find file path " + str(file_path))
             return False
 
+        filecmd = "file"
+        cmdpath = self.du.which(exe=filecmd)
+        # du.which returns the input cmd name if it can't find the cmd
+        if cmdpath is filecmd:
+            self.filecmdnotfound = True
+            return False
```

Comment:
Do this at the start of the process.  Print the error message then, and then rename this variable 'capture_core_files' and use it here.